sql/core/src/test/resources/sql-tests/inputs/subquery/in-subquery/in-joins.sql

```diff
@@ -6,8 +6,8 @@
 --  2. run with whole-stage-codegen, operator codegen or no codegen.
 
 --CONFIG_DIM1 spark.sql.autoBroadcastJoinThreshold=10485760
---CONFIG_DIM1 spark.sql.autoBroadcastJoinThreshold=-1,spark.sql.join.preferSortMergeJoin=true
---CONFIG_DIM1 spark.sql.autoBroadcastJoinThreshold=-1,spark.sql.join.preferSortMergeJoin=false
+--CONFIG_DIM1 spark.sql.autoBroadcastJoinThreshold=10485760,spark.sql.join.preferSortMergeJoin=true
+--CONFIG_DIM1 spark.sql.autoBroadcastJoinThreshold=10485760,spark.sql.join.preferSortMergeJoin=false
```

Comment:
cc @maropu

Reply:
BTW, @warrenzhu25 . Please describe in the PR description why you think this is wrong.

Reply:
@warrenzhu25 - Shuffled hash join will only be enabled with proper config value for `spark.sql.autoBroadcastJoinThreshold` and `spark.sql.shuffle.partitions`, and one side should be 3x smaller compared to the other side ([code](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/joins.scala#L368-L381)). I don't think test cases here satisfy the second condition (one side 3x smaller than the other side). Can you double check the query plan? Thanks.

Reply:
I saw from Jira description about this, maybe I misunderstood this Jira. @c21, could you help clarify more on this?

Reply:
@warrenzhu25 - I added a comment in JIRA. TLDR is I think only change config is not enough, we may need to change test input data, which is more involved and not sure whether it's worth the effort.

Reply:
Thanks for update. I'll close this.

Reply:
Thanks for pinging me, @dongjoon-hyun. IMO it might be worth adding a new directive (e.g., `--JOIN_TYPE`?)  for controlling physical join plans correctly via the hint logic in this test suite;
https://github.com/apache/spark/blob/e11a24c1ba5b0f3116b46a213443902165919da5/sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala#L70-L88